install.ps1

```diff
@@ -0,0 +1,31 @@
+# Create gengo directory in AppData/Roaming
+$gengoPath = "$env:APPDATA\gengo"
+New-Item -ItemType Directory -Force -Path $gengoPath -ErrorAction SilentlyContinue | Out-Null
+Write-Host "Installing gengo to $gengoPath"
+
+# Create and use temporary directory
+$tempDir = Join-Path ([System.IO.Path]::GetTempPath()) ([System.IO.Path]::GetRandomFileName())
+New-Item -ItemType Directory -Force -Path $tempDir -ErrorAction SilentlyContinue | Out-Null
+Push-Location $tempDir
+
+try {
+    # Download and extract gengo
+    $tempFile = "gengo-installer.tar.gz"
+    Invoke-WebRequest -Uri "https://github.com/spenserblack/gengo/releases/latest/download/gengo-x86_64-pc-windows-msvc.tar.gz" -OutFile $tempFile
+    tar -xzf $tempFile
```

Comment:
Nitpick: I prefer to keep the extension in Windows scripts.
```suggestion
    tar.exe -xzf $tempFile
```
(it also provides slightly better syntax highlighting)